extensions/anonymized-location/examples/collection.json

```diff
@@ -0,0 +1,34 @@
+{
+  "stac_version": "1.0.0-beta.1",
```

Comment:
should update to beta2

Reply:
Also, the JSON Schema should be updated to reflect recent changes we did to extension schemas.